tests/test_validators.py

```diff
@@ -72,6 +73,15 @@ def test_public_email_domain(self):
             self.assertIn('webmail_domain', self.webmail_form.errors)
             self.assertNotIn('not_webmail_domain', self.webmail_form.errors)
 
+    def test_public_email_domain_helper(self):
+        with self.app.test_request_context('/'):
+            self.assertEqual(is_public_email_domain(u'gmail.com'), True)
+            self.assertEqual(is_public_email_domain(u'i❤.ws'), False)
```

Comment:
Use a better known domain than i❤.ws for your tests. That domain is only for the IDN test. You could use contrasts such as `gmail.com` vs `google.com`.